dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/helpers/validate-csv-cells/index.ts

```diff
@@ -0,0 +1,85 @@
+import { UtilisationReportCsvRowData, UtilisationReportDataValidationError } from '@ukef/dtfs2-common';
+
+interface FacilityData {
+  baseCurrency: string;
+  facilityUtilisation: string;
+}
+
+/**
+ * validateUtilisationReportCells
```

Comment:
I'm finding the name of this and the other "validate cells" function a bit confusing:

```ts
validateUtilisationReportCsvCellData
validateUtilisationReportCells
```

Are we able to have clearer names to differentiate the two?